Result/AbstractResultsIterator.php

```diff
@@ -310,6 +310,24 @@ public function getDocumentScore()
     }
 
     /**
+    * Returns sort of current hit.
+    *
+    * @return array
+    */
+    public function getDocumentSort()
```

Comment:
I guess this follows `getDocumentScore()`. Later we will add getter for `id` too, so I think it would be nice to have single pattern.